### PR TITLE
04-onion-routing.md: Prepare for extra onion blob encoding.

### PR DESCRIPTION
**Pull request by ZmnSCPxj**

### CODE REVIEW

**Comment by ZmnSCPxj** [comment]
I used `per_hop_type` instead of agree upon `type`, but I feel `type` is imprecise as we also use the same term for message ID.

**Comment by ZmnSCPxj** [comment]
I have also named `per_hop_type` 0 to `normal_per_hop`.

File: 04-onion-routing.md [hunk 1]
```diff
@@ -161,22 +161,22 @@ and has the following structure:
 
 1. type: `hops_data`
 2. data:
-   * [`1`:`realm`]
+   * [`1`:`per_hop_type`]
```
Comment (Roasbeef) [inline 04-onion-routing.md]: Naming suggestion: packet type?

Comment (ZmnSCPxj) [reply 04-onion-routing.md]: That does seem better.

**Comment by t-bast** [comment]
Closing this old PR, superceded by #619

**Pull request closed without merging**